pop-benchmarker/build.gradle

```diff
@@ -21,5 +21,8 @@ android {
 
 dependencies {
     implementation "androidx.appcompat:appcompat:$rootProject.ext.appCompatVersion"
-    implementation "com.nimbusds:nimbus-jose-jwt:7.8"
+    implementation "androidx.constraintlayout:constraintlayout:$rootProject.ext.constraintLayoutVersion"
+    implementation(project(":common")) {
+        //transitive = false
```

Comment:
Remove / cleanup

Reply:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1068